www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -135,6 +135,87 @@
         ADD `login_attempts` INT(11) UNSIGNED DEFAULT NULL,
         ADD `blocking_time` BIGINT(20) UNSIGNED DEFAULT NULL"
     );
+
+    /**
+     * Add new UnifiedSQl broker output
+     */
+    $pearDB->beginTransaction();
+
+    $errorMessage = 'Unable to update cb_type table ';
+
+    $statement = $pearDB->query(
+        "UPDATE `cb_type` set type_name = 'Perfdata Generator (Centreon Storage) - DEPRECATED'
+            WHERE type_shortname = 'storage'"
+    );
+    $statement = $pearDB->query(
+        "UPDATE `cb_type` set type_name = 'Broker SQL database - DEPRECATED'
+            WHERE type_shortname = 'sql'"
+    );
```

Comment:
```suggestion
    $pearDB->query(
        "UPDATE `cb_type` set type_name = 'Perfdata Generator (Centreon Storage) - DEPRECATED'
            WHERE type_shortname = 'storage'"
    );
    $pearDB->query(
        "UPDATE `cb_type` set type_name = 'Broker SQL database - DEPRECATED'
            WHERE type_shortname = 'sql'"
    );
```
No need to store your query in a variable as you don't do anything with it